overall-equipment-effectiveness/workflows/Paper_Machine_Data_Simulator.yaml

```diff
@@ -0,0 +1,1969 @@
+resourceType: Flow
```

Comment:
Your Random Number node is going from 0 to 100; your description talks about 0 to 190.

<img width="1429" alt="Screen Shot 2020-09-25 at 6 21 29 PM" src="https://user-images.githubusercontent.com/881769/94320858-0689f500-ff5c-11ea-9d93-4ccaf73de387.png">

Reply:
I did this to force the status of the machine to run - I found a bug last week that was keeping it in a constant sku change. I have reverted the number back to 190.